dev-requirements.txt

```diff
@@ -14,3 +14,4 @@ ujson==1.35
 # dataclass serialization
 dacite==0.0.16
 pydash==4.7.3
+dvc==0.20.2
```

Comment:
If we're updating dev-requirements.txt, can we also tag in `requests>=2.20.0` to satisfy https://github.com/bmeg/bmeg-etl/network/alert/dev-requirements.txt/requests/open